asv_bench/benchmarks/frame_methods.py

```diff
@@ -1,13 +1,16 @@
-from .pandas_vb_common import *
 import string
+import numpy as np
+import pandas.util.testing as tm
+from pandas import (DataFrame, Series, MultiIndex, date_range, period_range,
+                    isnull, NaT)
 
-#----------------------------------------------------------------------
-# get_numeric_data
 
-class frame_get_numeric_data(object):
+class GetNumericData(object):
+
     goal_time = 0.2
 
     def setup(self):
+        np.random.seed(1234)
```

Comment:
You mentioned a module level setup function. Can we use that for the random seed to avoid repeating it for each benchmark?

Reply:
In my [comment](https://github.com/pandas-dev/pandas/pull/18500#issuecomment-347068970) I was actually referring to the `setup` function of each benchmark class instead of a module level `setup` function since the asv docs imply that the `setup` function of each class is run before each `time_*` function in that class.

Eventually I think we should transition to using 'standardized' data that could be initialized once at the module level.

Reply:
But in your comment you included this quote from the asv docs:

> You can also include a module-level setup function, which will be run for every benchmark within the module, prior to any setup assigned specifically to each function.

which I understood as a module level `def setup()` that is called before each benchmark. So I *think* that should work, but not sure.

> Eventually I think we should transition to using 'standardized' data that could be initialized once at the module level.

Yes, I agree we should try to do that more (at least for those cases where the data is not modified)